src/js/graph/graph.js

```diff
@@ -12,7 +17,11 @@ function Graph(id) {
     this._sequenceNodeTypes = { };
     this._spanContainerNodeTypes = { };
     this._nodeTypes = { };
-    this._graphImpl = new GraphImpl({ multigraph: true });
+    this._graphImpl = new GraphImpl({
+        directed: true,
+        multigraph: true,
```

Comment:
Why is `multigraph` and `compound` added here? I don't see a `.setParent` for `compound` and does the graph now have multiple edges between the same pair of nodes as `multigraph` would indicate?